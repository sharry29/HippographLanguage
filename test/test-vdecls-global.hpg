int a;
int b;
string c;

int main() {
  b = 40;
  a = b + 2;
  string c = "foo";
  print_int(a);
  print_int(b);
  print(c);

  return 0;
}
