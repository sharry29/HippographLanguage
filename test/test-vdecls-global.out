42
40
foo